test/e2e/fixtures/build-default/package.json

```diff
@@ -3,5 +3,11 @@
     "build": "tsdx build"
   },
   "name": "build-default",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@types/lodash": "4.14.165",
+    "@types/node": "14.14.7",
+    "lodash": "4.17.20",
+    "regenerator-runtime": "0.13.7"
```

Comment:
yarn 2 requires dependencies to be explicitly declared, so without the declarations here, they fail.  For example, `tsdx build` fails when it can't find `@types/node`, so that needs to be explicitly declared.

The hard pinning was for performance to make sure tests all use the same versions that were already pulled into local cache.  I was getting network failures from running multiple `yarn` processes in parallel, all trying to make many requests to npm.  It's possible the hard pinning can be removed since IIRC I started copying a seed `yarn.lock` into each stage directory before running `yarn`